CK3toEU4/Source/EU4World/Output/outWorld.cpp

```diff
@@ -145,6 +153,12 @@ void EU4::World::outputReligionIcons(const Configuration& theConfiguration, cons
 	interFaceDump.close();
 }
 
+void EU4::World::outputCultures(const Configuration& theConfiguration) const
```

Comment:
You only use theConfiguration for the output name, best to just take the output name as a parameter.